src/compose/testing/models_metaschema.xml

```diff
@@ -36,6 +36,9 @@
         <flag ref="id"/>
         <model>
             <field ref="field-1only"/>
+            <field>
+                <use-name>overloaded</use-name>
+            </field>
```

Comment:
Can you explain a little more in the comment or issue what this should be accomplishing? A `field` "by ref" that is `@ref`less should not be allowed and should not work, but I guess I would need to know more context to understand why this change is valid.

Reply:
when the JSON Schema is generated, the ref that is specified in the other overloaded fields doesn't exist causing an error. this field is so that the ref parameters in the other fields actually point to something

Reply:
I've removed that part and we can fix it with the other PR with the json schema testing